library/Makefile

```diff
@@ -5,7 +5,20 @@ CFLAGS	?= -O2
 WARNING_CFLAGS ?=  -Wall -W -Wdeclaration-after-statement
 LDFLAGS ?=
 
-LOCAL_CFLAGS = $(WARNING_CFLAGS) -I../include -D_FILE_OFFSET_BITS=64
+DIR_FOR_MBED_TLS_ENV=../library
+ifneq "$(wildcard $(DIR_FOR_MBED_TLS_ENV) )" ""
+    # Set include dirs for Mbed TLS test environment
+	INCLUDE_DIRS=-I../include
+else
+    # Set include dirs for Mbed OS test environment
+	INCLUDE_DIRS=-I.. -I../inc
```

Comment:
Why do we have `..` here, too?

Reply:
It is because `config.h` has been adjusted in Mbed OS. There is `#include "platform/inc/platform_mbed.h"`. We can remove `-I..` if only `test_config.h` is allowed for testing in Mbed OS.

Reply:
Ok, I don't mind leaving this as is for now.